src/tests/ftest/osa/osa_online_drain.py

```diff
@@ -28,28 +27,30 @@ def setUp(self):
         """Set up for test case."""
         super().setUp()
         self.dmg_command = self.get_dmg_command()
-        self.ior_flags = self.params.get("ior_flags", '/run/ior/iorflags/*')
-        self.ior_apis = self.params.get("ior_api", '/run/ior/iorflags/*')
+        self.daos_command = DaosCommand(self.bin)
```

Comment:
```suggestion
        self.daos_command = self.get_daos_command()
```
I believe you could do this.